e2e-tests/cypress/package.json

```diff
@@ -101,5 +101,8 @@
     "uniq-meta": "grep -r \"^// $META:\" cypress | grep -ow '@\\w*' | sort | uniq",
     "check": "eslint --ext .js,.ts . --quiet --cache",
     "fix": "eslint --ext .js,.ts . --quiet --fix --cache"
+  },
+  "dependencies": {
+    "cypress-real-events": "^1.14.0"
   }
```

Comment:
Should be fine here but let's move to `devDependencies` for consistency and use a fix version.

Reply:
done!